views/partials/account-navigation.ejs

```diff
@@ -10,6 +10,7 @@
         <%-profileData?.displayName || profileData?.username %>
       </div>
       <img
+        onerror='this.style.display="none"'
```

Comment:
I would prefer to add a condition around the `img` element checking if `profileData.country` or havinga placeholder icon